lib/core/network/router.js

```diff
@@ -189,7 +190,18 @@ class Router {
         cb(request);
       }
       else {
-        global.kuzzle.funnel.execute(mutatedRequest, (err, result) => cb(result));
+        global.kuzzle.funnel.execute(mutatedRequest, (err, result) => {
+          const _res = result || request;
+
+          if (err && ! _res.error) {
+            console.log('[Router] ERRRO R:', err);
```

Comment:
Use global.kuzzle.log.error function instead ?

Reply:
Yes, I was debugging and forgot to delete this console.log